Packs/Venafi/Integrations/VenafiV2/VenafiV2.py

```diff
@@ -0,0 +1,371 @@
+            url_suffix=url_suffix,
+            headers=headers
+        )
+
+        return certificate_details
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client) -> str:
+    """Tests API connectivity and authentication'
+
+    Returning 'ok' indicates that the integration works like it is supposed to.
+    Connection to the service is successful.
+    Raises exceptions if something goes wrong.
+
+    :type client: ``Client``
+    :param Client: client to use
+
+    :return: 'ok' if test passed, anything else will fail the test.
+    :rtype: ``str``
+    """
+
+    message: str = ''
+    try:
+        test_empty_args: Dict = {}
+        results = client._get_certificates(test_empty_args)
+        if results:
+            message = 'ok'
+    except DemistoException as e:
+        if 'Forbidden' in str(e) or 'Authorization' in str(e):
+            message = 'Authorization Error: make sure API Key is correctly set'
+        else:
+            raise e
+    return message
+
+
+def get_certificates_command(client: Client, args: dict[str, Any]) -> CommandResults:
+    """
+    Get all the certificates belong to a user.
+
+    Args:
+        client (Client): A Venafi client.
+        args (dict): The arguments for the command passed to the request.
+    Returns:
+        A CommandResult object with an outputs, raw_response and readable table, in case of a successful action.
+    """
+
+    outputs: dict[str, Any] = {}
+    response = client._get_certificates(args)
+    if response:
+        outputs = delete_links_from_response(response)
+
+    human_readable = []
+    certificates = outputs.get('Certificates', [])
+    for certificate in certificates:
+        certificate_details = get_human_readable_object(certificate)
+        human_readable.append(certificate_details)
+
+    markdown_table = tableToMarkdown('Venafi certificates', human_readable)
+
+    return CommandResults(
+        outputs_prefix=CONTEXT_OUTPUT_BASE_PATH,
+        outputs=outputs,
+        raw_response=response,
+        readable_output=markdown_table
+    )
+
+
+def get_certificate_details_command(client: Client, args: dict[str, Any]) -> CommandResults:
+    """
+    Get certificate details.
+
+    Args:
+        client (Client): A Venafi client.
+        args (dict): The arguments for the command passed to the request.
+    Returns:
+        A CommandResult object with an outputs, raw_response and readable table, in case of a successful action.
+    """
+
+    outputs: dict[str, Any] = {}
+    guid = args.get('guid', '')
+    response = client._get_certificate_details(guid)
+    if response:
+        outputs = response
+
+    human_readable = []
+    certificate_details = get_human_readable_object(outputs)
+    human_readable.append(certificate_details)
+
+    markdown_table = tableToMarkdown('Venafi certificate details', human_readable)
+
+    return CommandResults(
+        outputs_prefix=CONTEXT_OUTPUT_BASE_PATH,
+        outputs=outputs,
```

Comment:
```suggestion
        outputs=response if response else {},
```
If the response is `{}` when no response is returned from the API no need in the if and just enter the response here.